lib/models/importPersonasLock.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose';
```

Comment:
Let's check this model is needed.

Reply:
It's needed

Reply:
Feels like this is something that the persona service locks should handle?